css-canvas-medical/css/implementation.css

```diff
@@ -0,0 +1,79 @@
+.banner {
+  background-image: url('../images/implementation/implementation-header.jpg');
+  background-size: cover;
+  background-position: 0 -5rem;
+  height: 90vh;
+  width: 90vw;
+  margin: 7rem auto 10rem;
+}
+
+.overlay {
+  height: 100%;
+  background: rgba(115, 115, 115, 0.4);
+}
```

Comment:
I assume that these two are linked together? When using a `background-image`, you can use a `filter` property, and that property allows you to change the brightness/contrast/etc of an image. So you don't necessarily need to show an overlay to make it darker. Nothing wrong with this approach though!